src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Browser monitoring features
+metaDescription: 'The features associated with the New Relic browser monitoring Pro and Pro+SPA agents.'
+redirects:
+  - /docs/browser/new-relic-browser/browser-pro-features
+  - /docs/browser/new-relic-browser/browser-pro-features/intro-to-browser-pro-features
+---
+
+Check out our key browser monitoring features:
+
+* [**Summary**: View a summary of your browser's performance](/docs/browser/browser-monitoring/getting-started/browser-summary-page/)
+* [**Web vitals**: Learn what contributes to core web vitals](/docs/browser/new-relic-browser/guides/guide-to-monitoring-core-web-vitals/)
```

Comment:
This link doesn't work for me. I think the right one is [this](https://docs.newrelic.com/docs/journey-performance/guide-to-monitoring-core-web-vitals/).